LWOTC_WeaponPack/Config/XComLW_WeaponPack.ini

```diff
@@ -425,10 +596,12 @@ BULLPUP_COIL_IENVIRONMENTDAMAGE=5
 BULLPUP_COIL_UPGRADESLOTS=2
 
 AssaultRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilRifle_Base"
+BattleRifle_Coil_ImagePath="UILibrary_LW_Coilguns.InventoryArt.CoilRifle_Base"
```

Comment:
Did realitymachina's pack not have these assets? If not I'll need to put in an art request. Let me know